app/templates/teachers/teacher-student-view.jade

```diff
@@ -364,15 +365,22 @@ mixin breadcrumbs
                               .small!= marked(solution.description || '')
                           tr
                             td
-                              if view.levelStudentCodeMap[level.get('original')]
+                              - var codeMaps = view.levelStudentCodeMap[level.get('original')].filter(m => m)
+                              - var multipleCodes = false
```

Comment:
It shouldn't happen, but could sometimes in buggy scenarios, where there are more than two sessions. We can handle that, plus make this tighter, like this:
```suggestion
                              - var multipleCodes = codeMaps.length > 1
```